packages/integration-react/src/alpha.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { configApiRef, createApiFactory } from '@backstage/core-plugin-api';
+import {
+  createApiExtension,
+  createPlugin,
+} from '@backstage/frontend-plugin-api';
+import { ScmIntegrationsApi, scmIntegrationsApiRef } from './api';
+
+const ScmIntegrationsApiExtension = createApiExtension({
+  factory: createApiFactory({
+    api: scmIntegrationsApiRef,
+    deps: { configApi: configApiRef },
+    factory: ({ configApi }) => ScmIntegrationsApi.fromConfig(configApi),
+  }),
+});
+
+export default createPlugin({
```

Comment:
Does it make sense to export a plugin from here? Maybe create a plugin or add it to the default APIs list? Open for suggestions 🙂

Reply:
We're gonna need to dig into this quite a bit, kinda tricky 😅. I don't think plugin is great here but, but neither is an extensions override. Could ship as if for now and note as something we need to figure out